src/components/Room/LobbyView/LobbyView.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import Player from "../../../../server/classes/Player";
+import Room from "../../../../server/classes/Room";
+import IProfile from "../../../../server/interfaces/IProfile";
+import { useSocketRoom } from "../../../hooks";
+import useLocalStorage from "../../../hooks/useLocalStorage";
+import RoomService from "../../../services/RoomService";
+import { Layout, SectionTitle, Title } from "../../Common";
+import Button from "../../Common/Button/Button";
+import UserProfile from "../Id/UserProfile";
+
+export interface LobbyViewProps {
+    room: Room;
+}
+
+export function LobbyView(props: LobbyViewProps): JSX.Element {
+    const socket = useSocketRoom();
+    
+    const [playerId] = useLocalStorage("playerId");
+    
+    const [message, setMessage] = useState("");
+    const [messages, setMessages] = useState([]);
+    const [storageProfile] = useLocalStorage<IProfile>("profile");
+
+    const getRoomId = (): string => {
+        if(typeof window !== "undefined") {
+            return RoomService.getRoomIdFromUrl(window.location.href)
+        }
+        return "";
+    }
+    
+    const roomId = getRoomId();
+
+    useEffect(() => {
+        if(!socket) {
+            return;
+        }
+        
+        socket.on("receive-message-room", (message) => {
+            setMessages((prev) => [...prev, message]);
+        })
+    }, [socket])
+
+    const copyLinkToClipboard = () => {
+        if(typeof window !== "undefined") {
+            navigator.clipboard.writeText(window.location.href)
+        }
+    }
+
+    const startGame = () => {
+        if(props.room.creatorPlayerId === playerId) {
+            socket.emit('start-game');
+        }
+    }
+
+    const handleMessage = (e) => {
+        setMessage(e.currentTarget.value);
+    }
+    
+    const sendMessage = (e) => {
+        e.preventDefault();
+        receiveMessage({
+            username: storageProfile.username,
+            content: message
+        })
+        
+        if(message !== '') {
+            socket.emit("send-message-room", message, roomId);
+            setMessage('');
+        }
+    }
+    
+    const receiveMessage = (message) => {
+        setMessages(prev => [...prev, message])
+    }
+
+    return (
+        <Layout>
+            <div className="flex flex-wrap flex-auto justify-center md:space-x-32">
+                <div>
+                    <SectionTitle hintColor="text-yellow-light-yellow">Players</SectionTitle>
+                    {
+                        props.room?.players.map((player: Player) => <UserProfile key={player.id} player={player} creatorId={props.room.creatorPlayerId}/>)
```

Comment:
Ligne trop longue, la convention est de 100 char max. Le linter te gueule pas dessus ?

Reply:
Nope, linter doesn't care for long lines. Fixed and created issue to update linter.